app/(auth)/logout/LogoutButton.tsx

```diff
@@ -0,0 +1,30 @@
+import { cookies } from 'next/headers';
+import { deleteSessionByToken } from '../../../database/sessions';
+import styles from './LogoutButton.module.scss';
+
+export default function LogoutButtonClient() {
+  return (
+    <form>
+      <button
+        className={styles.logoutButton}
+        /* @ts-ignore */
+        formAction={async () => {
+          'use server';
```

Comment:
we may want to move this out to `app/(auth)/logout/actions.ts` because:

- then we have one example of a file with `'use server'` at the top
- inlining a server component inside of another component can be a risky pattern for security, as [it can lead to data exposure](https://twitter.com/tomus_sherman/status/1654198731313500174) - so maybe better for now to keep this separate, until we have a better understanding of this - and maybe even something like [static analysis or taint checking for preventing mistakes by students](https://twitter.com/karlhorky/status/1654223590928130049)

Reply:
For this to be implemented we also may need to update the prettier rules since at save it makes the use server to jump and be in parentesis on my experience. But yes ill update it